data/transport missions.txt

```diff
@@ -1867,6 +1872,297 @@ mission "Terraforming 5"
+			`	Amy introduces you to the four other people that she brought on board. Two of them are terraforming experts who helped Amy and Nolan with their research. The other two are representatives: one sent by the Republic Parliament in order to oversee the project, and another from Tundra.`
+			`	"Once you're ready, Captain, please bring us to <destination>. Tundra was a tropical world millions of years ago, but became cold after a cataclysmic event of some sort. There's a good chance that we may be able to nudge the planet back toward a warmer climate if we do this right."`
+					accept
+	
+	on complete
+		dialog 
+			`The journey to Tundra was spent by your passengers discussing the possible methods to terraform the planet. The Republic representative was none too pleased when Nolan, perhaps jokingly, suggested that we crash an asteroid into the planet.` 
+			`	Everyone except for the Tundra native begins to shiver after stepping out of your ship onto the planet, even while inside a protective dome. "Meet us in the spaceport bar in a few hours. We need to finalize our plans of how we're going to execute this experiment."`
+
+
+
+mission "Terraforming 10"
+	name "Terraforming Tundra"
+	description "Bring the Republic representative overseeing the terraforming project to <destination> where he will attempt to find an alternative solution to terraforming Tundra that does not involve an asteroid."
+	destination "Earth"
+	passengers 1
+	to offer
+		has "Terraforming 9: done"
+	
+	on offer
+		conversation
+			`Amy, Nolan, and the other two scientists tell you the known history of Tundra's climate. Millions of years ago, an asteroid impact or large volcanic eruption altered the planet's atmosphere. Now the planet no longer retains enough heat from its host star in order to stay warm.`
+			`	"Tundra's snow covered surface only exacerbates the problem, reflecting most of the star's light back into space." Nolan explains more. "This means that we need to both alter the atmosphere in order for it retain more heat and alter the surface in order for it to reflect less heat."
+			`	Amy rolls out a map of the entire planet onto the table. Different regions of the surface are color coded, and a large red spot in the planet's southern hemisphere sticks out to you.`
+			`	"Tundra has very low volcanic activity, but seismic mapping has lead us to this area." Amy circles the red spot with her finger. "Magma from the planet's mantle is building up under this area of the crust, which lucky for us is considerably thin. Still, it could be another 10,000 years before the magma finally erupts onto the surface in a supereruption, melting the snow around it and releasing carbon dioxide into the atmosphere. Tundra's crust also has an unusually high amount of serpentinite, which could release methane during the eruption and increase the amount of the star's energy that gets trapped in the atmosphere."`
```

Comment:
I'm not certain about the part that says, `10,000 years`. Is a numeric format acceptable, or should it be written out in words?